vpr/src/pack/greedy_clusterer.h

```diff
@@ -158,6 +246,25 @@ class GreedyClusterer {
     const std::unordered_set<AtomNetId>& is_global_;
 
     /// @brief Pre-computed logical block types for each model in the architecture.
-    std::map<const t_model*, std::vector<t_logical_block_type_ptr>> primitive_candidate_block_types_;
+    const std::map<const t_model*, std::vector<t_logical_block_type_ptr>> primitive_candidate_block_types_;
+
+    /// @brief The verbosity of log messages produced by the clusterer.
+    ///
+    /// Numbers larger than 2 will print info on the status of the packing for
+    /// each molecule.
+    const int log_verbosity_;
+
+    /* Does the atom block that drives the output of this atom net also appear as a   *
```

Comment:
Get rid of trailing *'s. (old style by me, not good anymore).
Make Doxygen style.